generated/nidaqmx/_base_interpreter.py

```diff
@@ -0,0 +1,1963 @@
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    def create_tedsai_voltage_chan_with_excit(
+            self, task, physical_channel, name_to_assign_to_channel,
+            terminal_config, min_val, max_val, units, voltage_excit_source,
+            voltage_excit_val, custom_scale_name):
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    def create_tedsairtd_chan(
+            self, task, physical_channel, name_to_assign_to_channel, min_val,
+            max_val, units, resistance_config, current_excit_source,
+            current_excit_val):
+        raise NotImplementedError
+
+    @staticmethod
+    @abc.abstractmethod
+    def create_watchdog_timer_task(
+            self, device_name, session_name, timeout, lines, exp_state,
+            initialization_behavior):
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    def create_watchdog_timer_task_ex(
+            self, device_name, session_name, timeout,
+            initialization_behavior):
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    def delete_network_device(self, device_name):
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    def delete_saved_global_chan(self, channel_name):
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    def delete_saved_scale(self, scale_name):
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    def delete_saved_task(self, task_name):
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    def device_supports_cal(self, device_name):
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    def disable_ref_trig(self, task):
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    def disable_start_trig(self, task):
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    def disconnect_terms(self, source_terminal, destination_terminal):
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    def export_signal(self, task, signal_id, output_terminal):
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    def get_ai_chan_cal_cal_date(self, task, channel_name):
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    def get_ai_chan_cal_exp_date(self, task, channel_name):
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    def get_analog_power_up_states(
+            self, device_name, channel_name, channel_type):
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    def get_analog_power_up_states_with_output_type(
+            self, channel_names, array_size):
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    def get_arm_start_trig_timestamp_val(self, task):
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    def get_arm_start_trig_trig_when(self, task):
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    def get_auto_configured_cdaq_sync_connections(self) -> str:
```

Comment:
Ok, well, it's a start 😅

Partial type annotations are better than nothing (as long as they're correct).